<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TortoiseDialog</class>
 <widget class="QWidget" name="TortoiseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a string here, and the same string in the Client Hash field on the Client computer.&lt;/p&gt;&lt;p&gt;This string will usually be a hash of some data on production software. Here, anything will do.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Server Hash</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverHash">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>161</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="serverHashButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>162</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Listen to this</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientHash">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>201</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a string here, and the same string in the Server Hash field on the Server computer.&lt;/p&gt;&lt;p&gt;This string will usually be a hash of some data on production software. Here, anything will do.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Client Hash</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clientHashButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>202</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Monitor this</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>491</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter any string here, and the same string
on the other side in the other field
Then send a message</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a string here, and the same string in the Server Hash field on the Server computer.&lt;/p&gt;&lt;p&gt;This string will usually be a hash of some data on production software. Here, anything will do.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Message:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="message">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>271</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMessageButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
